test_scripts/clio_223.js

```diff
@@ -0,0 +1,38 @@
+/*Script provided by Nathan Nichols*/
```

Comment:
Imho, not a great name for the script - `clio_223`. i suppose it is referring to the issue 223 but i would try call it something meaningful instead like `requestAccountChannels` for example. 

Another thing is - i'd probably not put these two scripts in the codebase at all. At the very least not until everyone agree we want to have those in the codebase and they should be in javascript. We already have one uberscript written in python called `test.py`. 

@cjcobb23 what do you think about tests scripts in the repo?